include/geos/operation/polygonize/PolygonizeGraph.h

```diff
@@ -67,20 +69,18 @@ namespace polygonize { // geos::operation::polygonize
  *
  */
 class GEOS_DLL PolygonizeGraph: public planargraph::PlanarGraph {
-
-public:
-
+ public:
 	/**
 	 * \brief
 	 * Deletes all edges at a node
 	 */
-	static void deleteAllEdges(planargraph::Node *node);
+	void deleteAllEdges(planargraph::Node *node);
```

Comment:
Was this intentional ? Was it not a static method in JTS ?

Reply:
Overhead arguments aside, I think keeping `static` shows intent more clearly.